src/snowflake/cli/api/utils/models.py

```diff
@@ -1,9 +1,15 @@
+from __future__ import annotations
+
 import os
+from typing import Any, Dict
 
 
-class EnvironWithDefinedDictFallback(dict):
+class EnvironWithDefinedDictFallback(Dict):
     def __getattr__(self, item):
-        return self[item]
+        try:
+            return self[item]
+        except KeyError:
+            raise AttributeError
```

Comment:
Shouldn't we say something about which attribute it was? Python itself does something like '<object> has no atttribute <name>'

Reply:
AttributeError only takes an error as input. That will at least help identify what attribute value it is. does this address your concern?
for now, it's only used with hasattr() which only returns false if there is attributerror.